bundles/org.eclipse.osgi.tests/src/org/eclipse/osgi/tests/resource/BasicTest.java

```diff
@@ -457,12 +462,12 @@ private void assertNotRequirementMatches(Requirement requirement, Capability cap
 		assertFalse("Requirement matches capability", ((BundleRequirement) requirement).matches((BundleCapability) capability));
 	}
 
-	private void assertNotNull(Capability capability) {
-		Assert.assertNotNull("Null capability", capability);
+	private void assertNotNullCapabilities(Capability capability) {
+		assertNotNull("Null capability", capability);
 	}
```

Comment:
IMHO it would also be suitable to just inline `assertNotNull(capability)`.
At least from my perspective these comments are often not that helpful because in most cases one has to check the code anyways.

Reply:
Fully agreed. I have simply removed the custom `assertNotNull` methods in 752ffd9.